edit_book.php

```diff
@@ -0,0 +1,44 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code directly uses raw SQL queries with user input, which is vulnerable to SQL injection attacks.

Recommendations:
- **Use Prepared Statements**: Utilize prepared statements for the `SELECT` and `UPDATE` queries to prevent SQL injection.
- **Example**:
  ```php
  $stmt = $connection->prepare("SELECT * FROM books WHERE id = ?");
  $stmt->bind_param("i", $id);
  $stmt->execute();
  $result = $stmt->get_result();
  ```